src/containers/my-cooperations/cooperation-notes/create-or-edit-note/CreateOrEditNote.tsx

```diff
@@ -62,10 +85,25 @@ const CreateOrEditNote = ({
       sx={styles.container}
     >
       <Box sx={styles.header}>
-        <Avatar />
-        <Typography variant={TypographyVariantEnum.Subtitle2}>
-          {userNameMocked}
-        </Typography>
+        {loading ? (
+          <Loader size={20} />
+        ) : (
+          <>
+            <Avatar
+              src={
+                photo
+                  ? `${import.meta.env.VITE_APP_IMG_USER_URL}${photo}`
+                  : undefined
+              }
+              sx={styles.accountIcon}
+            >
+              {firstName && lastName && `${firstName[0]}${lastName[0]}`}
+            </Avatar>
+            <Typography variant={TypographyVariantEnum.Subtitle2}>
+              {firstName && lastName && `${firstName} ${lastName}`}
```

Comment:
Same here. And it is pretty similar with following code snippet:

```jsx
{firstName && lastName && `${firstName[0]} ${lastName[0]}`}
```

Please consider reusing it

Reply:
Fixed